src/components/strategies/portfolio/PortfolioMobileCard.tsx

```diff
@@ -40,7 +42,12 @@ export const PortfolioMobileCard: FC<Props> = ({
 
   if (href) {
     return (
-      <Link to={href} className={wrapperClasses}>
+      <Link
```

Comment:
When should we use `Link` over `A` ?

Reply:
A is just a wrapper around the native anchor element with some defaults. Use this for any external links.
Link is the react-router component for internal navigation.

I didnt get type-safety to work properly with the combo component we had.